spec/models/user_spec.rb

```diff
@@ -5,4 +5,71 @@
   it { should have_many :answers }
   it { should validate_presence_of :email }
   it { should validate_presence_of :password }
+
+  describe '.find_for_oauth' do
+    let!(:user) { create(:user) }
```

Comment:
How is it possible rubocop don't warning you about `let!`?

Reply:
No

Reply:
Please tune it in write a way to warn you.